app/forms/declaration/DeclarationHolder.scala

```diff
@@ -34,7 +34,7 @@ object DeclarationHolder {
       text()
         .verifying("supplementary.declarationHolder.authorisationCode.invalid", isContainedIn(HolderOfAuthorisationCode.all.map(_.value)))
     ),
-    "eori" -> optional(text().verifying("supplementary.eori.error", lengthInRange(1)(17) and isAlphanumeric))
+    "eori" -> optional(text().verifying("supplementary.eori.error.format", lengthInRange(1)(17) and isAlphanumeric))
```

Comment:
Similar here, we should start using new `isValidEORI` validation

Reply:
Done

Reply:
Done